tools/acle_template.tex

```diff
@@ -0,0 +1,148 @@
+% Copyright (c) 2014--2021, John MacFarlane
+% Based on default.latex template released with Pandoc (www.pandoc.org).
+% License: BSD 3-clause
+% See `data/templates` in `COPYRIGHT` at https://github.com/jgm/pandoc
+
+\documentclass[$if(fontsize)$$fontsize$,$endif$$if(papersize)$$papersize$paper,$endif$$for(classoption)$$classoption$$sep$,$endfor$]{$documentclass$}
+\usepackage{xcolor}
+\usepackage{hyperref}
+\usepackage{longtable,booktabs}
+\urlstyle{same}  % don't use monospace font for urls
+\setlength{\emergencystretch}{3em}  % prevent overfull lines
+\providecommand{\tightlist}{%
+  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}
+\usepackage{fancyhdr}
+\usepackage{etoolbox}
+
+$if(geometry)$
+\usepackage[$for(geometry)$$geometry$$sep$,$endfor$]{geometry}
+$endif$
+
+\makeatletter
+  \patchcmd{\f@nch@foot}{\rlap}{\color{$if(fontcolor)$$fontcolor$$else$black$endif$}\rlap}{}{}
+  \patchcmd{\footrule}{\hrule}{\color{$if(fontcolor)$$fontcolor$$else$black$endif$}\hrule}{}{}
+\makeatother
+
+\pagestyle{fancy}
+
+\fancyhf{}
+$if(footertext)$\fancyfoot[L]{\small \emph{$footertext$}}$endif$
+\fancyfoot[R]{\small \thepage}
+\renewcommand{\footrulewidth}{0.4pt}
+
+\fancypagestyle{plain}{%
+  \fancyhf{}
+  $if(footertext)$\fancyfoot[L]{\small \emph{$footertext$}}$endif$
```

Comment:
We always want the copyright statement, no need to wrap it in a `if` statement.

Reply:
That's just how the template recognises yaml values. I don't think removing the if statement would do.

Reply:
Variables like `$body` and `$classoption` don't seem to need the `$if`? Or are these special cases?

Reply:
classoption is wrapped in a for loop instead and body is just the content of the input file. I'm not completely familiar with the syntax, but that I noticed that as pattern - a yaml exclusive variable can't be referenced unless wrapped in some sort of functionality that won't run if no variable has been specified.
In any case, when I tried to use the variable without an if statement the copyright text wasn't being printed out.

Reply:
Thank you for checking, @ValeriaDaneva  - you convinced me :)